sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/projects/aio/operations/_operations.py

```diff
@@ -0,0 +1,1815 @@
+    @distributed_trace_async
+    async def create_project(
+        self,
+        project_name: str,
+        body: "_models.CreateProjectOptions",
+        **kwargs: Any
+    ) -> "_models.ProjectMetadata":
+        """Create or update a project.
+
+        Create or update a project.
+
+        :param project_name: The name of the project to use.
+        :type project_name: str
+        :param body: Parameters needed to create the project.
+        :type body: ~azure.ai.language.questionanswering.projects.models.CreateProjectOptions
+        :keyword api_version: Api Version. The default value is "2021-10-01". Note that overriding this
+         default value may result in unsupported behavior.
+        :paramtype api_version: str
+        :return: ProjectMetadata
+        :rtype: ~azure.ai.language.questionanswering.projects.models.ProjectMetadata
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        cls = kwargs.pop('cls', None)  # type: ClsType["_models.ProjectMetadata"]
+        error_map = {
+            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
+        }
+        error_map.update(kwargs.pop('error_map', {}))
+
+        api_version = kwargs.pop('api_version', "2021-10-01")  # type: str
+        content_type = kwargs.pop('content_type', "application/json")  # type: Optional[str]
+
+        json = self._serialize.body(body, 'CreateProjectOptions')
+
+        request = build_question_answering_projects_create_project_request(
+            project_name=project_name,
+            api_version=api_version,
+            content_type=content_type,
+            json=json,
+            template_url=self.create_project.metadata['url'],
+        )
+        path_format_arguments = {
+            "Endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, 'str', skip_quote=True),
+        }
+        request.url = self._client.format_url(request.url, **path_format_arguments)
+
+        pipeline_response = await self._client._pipeline.run(request, stream=False, **kwargs)
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200, 201]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            error = self._deserialize.failsafe_deserialize(_models.ErrorResponse, pipeline_response)
+            raise HttpResponseError(response=response, model=error)
+
+        if response.status_code == 200:
+            deserialized = self._deserialize('ProjectMetadata', pipeline_response)
+
+        if response.status_code == 201:
+            deserialized = self._deserialize('ProjectMetadata', pipeline_response)
+
+        if cls:
+            return cls(pipeline_response, deserialized, {})
+
+        return deserialized
+
+    create_project.metadata = {'url': '/query-knowledgebases/projects/{projectName}'}  # type: ignore
+
+
+    @distributed_trace_async
+    async def delete_project(
```

Comment:
Is this the function that needs to be regenerated as an LRO? Has that swagger fix been merged?

Reply:
all functions that has LROs need to be re-generated
delete, export, update knowledge source, ..
and yes the new swagger update is released (but not merged)
and i did the regen